models/Product.js

```diff
@@ -1,23 +1,36 @@
 module.exports = function(sequelize, DataTypes) {
   var Product = sequelize.define("Product", {
-    title: {
+    name: {
       type: DataTypes.STRING,
       allowNull: false,
       validate: {
         len: [1]
       }
     },
-    body: {
-      type: DataTypes.TEXT,
+    sku: {
+      type: DataTypes.NUMBER,
```

Comment:
@Marifer-Enriquez I think this field should be a `STRING`.

Reply:
done!